common/lib/xmodule/xmodule/course_module.py

```diff
@@ -774,6 +774,30 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    learning_info = List(
+        display_name=_("Course Learning Information"),
+        help=_("Specify what student can learn from the course."),
+        default=[''],
+        scope=Scope.settings
+    )
+
+    instructor_info = Dict(
+        display_name=_("Course Instructor"),
+        help=_("Enter the details for Course Instructor"),
+        default={
```

Comment:
I think the difference is that with the grading policy, those default values are actually used. In your case, the default values would not make sense to be displayed outside of Studio.

Please work with @douglashall to determine what to do here.